src/containers/buttons/AddTestButton.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react';
+import {connect} from "react-redux";
+import {bindActionCreators} from 'redux';
+import {addTestToProject, clearTests, hideUrlInput, clearUrlsList, hideTestNameInput, initTestName} from "../../actions"
+
+class AddTest extends Component {
+
+  addTestToProject() {
+    if(this.props.test.length > 0 && this.props.urls.length > 0) {
+      document.querySelectorAll(".block").forEach((block) => {
```

Comment:
Is this seperate functionality from react app?

Reply:
I don't quite understand, what do you mean by that?